training/distributed_training/pytorch/model_parallel/gpt2/smp-train-gpt-simple.ipynb

```diff
@@ -307,18 +305,27 @@
     "    \"num_kept_checkpoints\": 5,\n",
     "    \"checkpoint_freq\": 200,\n",
     "    \"logging_freq\": 1,\n",
-    "    \"save_final_full_model\": 0,\n",
-    "    \"manual_partition\": 1,\n",
-    "    \"skip_full_optimizer\": 1,\n",
-    "    \"shard_optimizer_state\": 1,\n",
-    "    \"activation_checkpointing\": 1,\n",
-    "    \"activation_strategy\": \"each\",\n",
-    "    \"optimize\": \"speed\",\n",
     "    \"use_wiki_data\": 1,\n",
     "    # below flag loads model and optimizer state from checkpoint_s3_uri\n",
     "    # 'load_partial': 1,\n",
     "}\n",
     "\n",
+    "# Add parameters required by SMP config.\n",
```

Comment:
Why are these separated like this from above set?

Reply:
This was done based on the suggestion to mark which parameters correspond to SMP config: https://github.com/aws/amazon-sagemaker-examples/pull/3311#discussion_r878472254